config/routes.rb

```diff
@@ -57,4 +57,10 @@
   resources :attendances, controller: 'attendance/attendances', only: [:update]
   resources :event_types, controller: 'attendance/event_types', except: [:show]
 
+  #### Org Chart ###############################################################
+  resources :org_chart, controller: 'org_chart', only: [:index]
+  get 'org_chart_admin' => 'org_chart#admin', as: 'org_chart_admin'
+  post 'org_chart_create' => 'org_chart#create', as: 'org_chart/create'
+  post 'org_chart_remove' => 'org_chart#remove', as: 'org_chart/remove'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.